homeassistant/components/sensibo/switch.py

```diff
@@ -143,37 +113,77 @@ def is_on(self) -> bool | None:
 
     async def async_turn_on(self, **kwargs: Any) -> None:
         """Turn the entity on."""
-        params = build_params(self.entity_description.command_on, self.device_data)
-        result = await self.async_send_command(
-            self.entity_description.command_on, params
-        )
-
-        if result["status"] == "success":
-            setattr(self.device_data, self.entity_description.remote_key, True)
-            self.async_write_ha_state()
-            return await self.coordinator.async_request_refresh()
-        raise HomeAssistantError(
-            f"Could not execute {self.entity_description.command_on} for device {self.name}"
-        )
+        if self.entity_description.key == "timer_on_switch":
```

Comment:
Why don't we have different target functions per description?

Reply:
Isn't that what we have here?

Reply:
I mean that the target function should be put on the description so we don't need a check here.